dataprofiler/data_readers/avro_data.py

```diff
@@ -1,6 +1,6 @@
 """Contains class for saving and loading spreadsheet data."""
 from io import BytesIO, StringIO
-from typing import Any, Dict, List, Optional, Union, cast
+from typing import Any, Dict, List, Optional, Union
```

Comment:
the proper way in the other func for a boolean which checks type is to return a TypeGuard

Reply:
no need to cast now